src/storage/files.c

```diff
@@ -323,6 +323,11 @@ typedef struct pw_cb_data
 
 static int password_callback(char* buf, int bufsiz, int verify, void* cb_tmp)
 {
+    if(0 == buf || bufsiz <= 0 || 0 == cb_tmp)
```

Comment:
Why you you forbid `0 == cb_tmp` ?

Reply:
Better move the checks on `buf` and `bufsiz` below into `if(password not_eq 0) { `
because they are needed only in this case.